gitlab/v4/objects/projects.py

```diff
@@ -85,6 +87,17 @@ class GroupProjectManager(ListMixin, RESTManager):
     )
 
 
+class ProjectAudit(RESTObject):
+    _id_attr = "id"
+
+
+class ProjectAuditManager(ListMixin, RESTManager):
```

Comment:
Based on the upstream docs, this endpoint supports both single + all events endpoint. We have a convenience `RetrieveMixin` for endpoitns that support both .get() and .list():

```suggestion
class ProjectAuditManager(RetrieveMixin, RESTManager):
```